src/components/TodoElement/TodoElement.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { deleteTodoElement, toggleTodoElement } from "../../actions/actions";
+import { useDispatch } from "react-redux";
+import { Checkbox } from "antd";
+
+const TodoElement = ({ todo }) => {
+  console.log("render TodoElement");
+  const dispatch = useDispatch();
+
+  const onDelete = (id) => {
+    dispatch(deleteTodoElement(id));
+  };
+
+  const toggle = (id) => {
+    dispatch(toggleTodoElement(id));
+  };
+
+  return (
+    <div className="list-item">
+      <Checkbox checked={todo.checked} onClick={() => toggle(todo.id)} />
+      <div className={todo.checked ? "toggle" : ""} key={todo.id}>

```

Comment:
read about classnames npm package for this purpose, in this case it is enought, but in the nearest furure you will have more complex cases